src/contexts/applicationContext.tsx

```diff
@@ -42,18 +45,18 @@ function ApplicationProvider({ children }: ApplicationProviderProps) {
 
   useEffect(() => {
     invoke<MacroData>('get_macros')
-      .then((res) => {
+      .then((res: { data: SetStateAction<Collection[]> }) => {
         setCollections(res.data)
         setInitComplete(true)
       })
-      .catch((e) => {
+      .catch((e: string) => {
         error(e)
         toast({
           title: 'Error loading macros',
           description:
             'Unable to load macros, please re-open the app. If that does not work, please contact us on Discord.',
           status: 'error',
-          duration: 2000,
+          duration: 10000,
           isClosable: true
```

Comment:
If we're asking them to re-open the app, should we really be automatically closing this toast at all?

Reply:
I was thinking whether error toasts should have an autoclose at all, might be better to remove them all from all error toasts imho

Reply:
Tweaked here: [4529e09](https://github.com/WootingKb/wooting-macros/pull/194/commits/4529e09bffb2f40916220f31ac48a83d521aa7ee)